react/features/base/conference/middleware.any.js

```diff
@@ -132,7 +129,7 @@ function _conferenceFailed({ dispatch, getState }, next, action) {
     case JitsiConferenceErrors.CONFERENCE_DESTROYED: {
         const [ reason ] = error.params;
 
-        dispatch(showErrorNotification({
+        dispatch(showWarningNotification({
```

Comment:
We might need to trigger the readyToClose event for the iframe API, see below for an example of when we do that.

Reply:
Thanks `The meeting has been terminated` is now sent.